tests/test_expression.py

```diff
@@ -44,13 +45,21 @@
     ("0 | 1", 1),
     ("1 | 1", 1),
     ("1 | 2", 3),
```

Comment:
```suggestion
    ("1 | 2", 3),
    ("1 | 2 | 4", 7),
```
Chaining multiple bitwise operators  ("1 | 2 | 4", 7) seems to fail...

Reply:
Fixed, I also fixed the precedence of the `&` operator which seemed to be wrong.